x-pack/metricbeat/tests/integration/otel_test.go

```diff
@@ -146,6 +146,152 @@ func assertMonitoring(t *testing.T, port int) {
 	require.Equal(t, http.StatusNotFound, r.StatusCode, "incorrect status code")
 }
 
+func TestMetricbeatOTelReceiverE2E(t *testing.T) {
+	integration.EnsureESIsRunning(t)
+
+	host := integration.GetESURL(t, "http")
+	user := host.User.Username()
+	password, _ := host.User.Password()
+
+	namespace := strings.ReplaceAll(uuid.Must(uuid.NewV4()).String(), "-", "")
+
+	mbReceiverIndex := "logs-integration-mbreceiver-" + namespace
+	mbIndex := "logs-metricbeat-mb-" + namespace
+
+	type options struct {
+		Index    string
+		ESURL    string
+		Username string
+		Password string
+	}
+
+	cfg := `receivers:
+  metricbeatreceiver:
+    metricbeat:
+      modules:
+       - module: system
+         enabled: true
+         period: 1s
+         processes:
+          - '.*'
+         metricsets:
+          - cpu
+    output:
+      otelconsumer:
+    logging:
+      level: info
+      selectors:
+        - '*'
+    queue.mem.flush.timeout: 0s
+exporters:
+  debug:
+    use_internal_logger: false
+    verbosity: detailed
+  elasticsearch/log:
+    endpoints:
+      - {{.ESURL}}
+    compression: none
+    user: admin
+    password: testing
```

Comment:
Why is user/password hardcoded here but not [below](https://github.com/elastic/beats/pull/45427/files#diff-75d90ff76c3b9e75e085c5a9e34b5bd645a92f93f43d1d913da15ad534b3d85bR246-R247) in Metricbeat config?